Libraries/Opc.Ua.Server/Subscription/MonitoredItem.cs

```diff
@@ -1873,6 +1959,9 @@ private void QueueOverflowHandler()
         private IAggregateCalculator m_calculator;
         private bool m_triggered;
         private bool m_resendData;
+        HashSet<string> FilteredRetainConditionIds = null;
```

Comment:
shoudn´t this be private and m_filteredRetainConditionIds

Reply:
Addressed